scripts/test.js

```diff
@@ -4,6 +4,27 @@ require("@app/config/env");
 const { execSync, spawnSync: rawSpawnSync } = require("child_process");
 const concurrently = require("concurrently");
 
+// Dear graphile-migrate, please treat the test DB as if it were the shadow DB
+process.env.SHADOW_DATABASE_URL = process.env.TEST_DATABASE_URL;
+
+// Signal to graphile-migrate scripts that we're in the tests
+process.env.IN_TESTS = "1";
+process.env.NODE_ENV = "test";
+
+const cmdArgs = process.argv;
```

Comment:
I changed this file and the way how arguments are handled to add the `--delay` argument. It should work fine, but I am not sure if this is the preferred way. Otherwise we could use [minimist](https://www.npmjs.com/package/minimist) package to handle command line arguments in a slightly more sophisticated way.

Reply:
This is fine for now. No need for the extra weight of minimist for a couple flags.